subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/screen/ScreenChecks.kt

```diff
@@ -29,6 +29,8 @@ open class StrictScreenChecks<out T : Screen>(
 
 private fun Screen.checkRootId() {
     if (this.rootId != UNKNOWN_ROOT_ID) {
+        // TODO: lost interaction context
```

Comment:
Prefer to see the comment as a KDOC for this function

Reply:
Fixed.

I want to discuss this limitation in another PR. 
I have ideas on how to support it.